util/git/client.go

```diff
@@ -341,6 +341,19 @@ func (m *nativeGitClient) Init() error {
 		Name: git.DefaultRemoteName,
 		URLs: []string{m.repoURL},
 	})
+	if err != nil {
+		return err
+	}
+
+	// set automatic pack limit if specified
+	if countStr := os.Getenv(common.EnvGitAutopackLimit); countStr != "" {
+		if _, err := strconv.Atoi(countStr); err != nil {
+			panic(fmt.Sprintf("Invalid value in %s env variable: %v", common.EnvGitAutopackLimit, err))
```

Comment:
should we panic? maybe warning will be enough?

Reply:
Ey, I prefer panic because:

- Consistency: It is the same approach as `ARGOCD_GIT_ATTEMPTS_COUNT` for  consistency but also cause for me make sense to panic: 
- Visibility: A Warning hide the bad configuration making it harder to identify the issue.

Wdyt?

Reply:
Yeah, i think it makes sense for me. Especially if it is already implemented in such way in other place